greenback-kit-core/src/main/java/com/greenback/kit/model/AutoExport.java

```diff
@@ -0,0 +1,98 @@
+package com.greenback.kit.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class AutoExport extends GreenbackObject{
+    private String userId;
+    private String accountingAccountId;
+    private AutoExportRequest parameters;
+    private AutoExportFrequency frequency;
+    private AutoExportState state;
+
```

Comment:
Are we missing createdAt and updatedAt?

Reply:
I am extending GreenbackObject, so those are pulled in from that